tests/test_syslog.robot

```diff
@@ -0,0 +1,140 @@
+*** Settings ***
+
+Documentation       This suite is for testing syslog function of Open BMC.
+
+Resource            ../lib/rest_client.robot
+Resource            ../lib/utils.robot
+
+*** Variables ***
+${IP_ADDRESS}              9.109.116.67
+${PORT}                    514
+${INVALID_IP_ADDRESS}      a.ab.c.d
+${INVALID_PORT}            abc
+
+*** Test Cases ***
+
+Get all Syslog settings
+    [Documentation]     ***GOOD PATH***
+    ...                 This testcase is to get all syslog settings from
+    ...                 open bmc system.\n
+
+    ${resp} =    OpenBMC Get Request    /org/openbmc/LogManager/rsyslog
+    Should Be Equal As Strings    ${resp.status_code}    ${HTTP_OK}
+    ${jsondata}=    To Json    ${resp.content}
+    Should Contain     ${jsondata['data']}    ipaddr
```

Comment:
These three test cases can be clubbed together under Get all Syslog Settings testcase

Get ip address of Syslog setting
Get port address of Syslog setting
Get status of Syslog setting